packages/expo-updates/e2e/setup/project.ts

```diff
@@ -226,6 +226,13 @@ async function copyCommonFixturesToProject(
   // copy .prettierrc
   await fs.copyFile(path.resolve(repoRoot, '.prettierrc'), path.join(projectRoot, '.prettierrc'));
 
+  // Copy react-native patch
+  await fs.mkdir(path.join(projectRoot, 'patches'));
+  await fs.copyFile(
+    path.resolve(repoRoot, 'patches', 'react-native+0.83.0-nightly-20251030-26ad9492b.patch'),
+    path.join(projectRoot, 'patches', 'react-native+0.83.0-nightly-20251030-26ad9492b.patch')
+  );
```

Comment:
```suggestion
  // FIXME: when we remove the patch, also revert the `RCT_USE_PREBUILT_RNCORE=0` change
  await fs.mkdir(path.join(projectRoot, 'patches'));
  const patchFile = await glob('react-native+*.patch', { cwd: path.join(repoRoot, 'patches'), absolute: true });
  await fs.copyFile(
    patchFile,
    path.join(projectRoot, 'patches', path.basename(patchFile)
  );
```

use glob to search for patch file, that to reduce maintenance cost from bumping nightlies. that code here may not 100% correct. please help to double check that